addon/components/nrg-new-features/component.js

```diff
@@ -0,0 +1,43 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { alias, bool } from '@ember/object/computed';
+import { task } from 'ember-concurrency';
+import layout from './template';
+
+export default Component.extend({
+  layout,
+
+  whatsNewService: service('whats-new'),
+
+  isOpen: bool('content'),
+
+  headerText: alias('whatsNewService.headerText'),
+
+  primaryButtonText: alias('whatsNewService.primaryButtonText'),
+  secondaryButtonText: alias('whatsNewService.secondaryButtonText'),
+
+  checkContent: task(function*() {
+    let content = yield this.get('whatsNewService.getContent').perform();
```

Comment:
The method in the whats-new service does not have to be a concurrency task.
That prevents it from being able to return a synchronous value.
If you make it just a function `this.whatsNewService.getContent()` then it could be overridden with an `async` function by the end-user.
If a concurrency task is absolutely needed, then the async function can call a concurrency task, but it will likely not be required.